client/components/jetpack/connection-health/index.tsx

```diff
@@ -30,15 +29,11 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		( state ) => !! isSiteAutomatedTransfer( state, siteId )
 	);
 
-	const [ isErrorCheckJetpackConnectionHealth, setIsErrorCheckJetpackConnectionHealth ] =
-		useState( false );
-
-	const { data: jetpackConnectionHealth, isLoading: isLoadingJetpackConnectionHealth } =
-		useCheckJetpackConnectionHealth( siteId, {
-			onError: () => {
-				setIsErrorCheckJetpackConnectionHealth( true );
-			},
-		} );
+	const {
+		data: jetpackConnectionHealth,
+		isLoading: isLoadingJetpackConnectionHealth,
+		isError: isErrorCheckJetpackConnectionHealth,
```

Comment:
`isError` is provided by RQ, there's no need to sync state to the `useState` hook.

Reply:
Nice cleanup 👍